src/main/java/copado/onpremise/service/git/GitSession.java

```diff
@@ -1,10 +1,14 @@
 package copado.onpremise.service.git;
 
-import java.nio.file.Path;
+import lombok.Data;
+import org.eclipse.jgit.api.Git;
 
+import java.nio.file.Path;
 
-public interface GitSession {
```

Comment:
This interface shouldn't be deleted because of the same reason as in `Branch`  one